packages/combo-box/test/lazy-loading.test.js

```diff
@@ -473,6 +473,8 @@ describe('lazy loading', () => {
           await aTimeout(0);
 
           comboBox._scrollIntoView(0);
+          // Wait for the async data provider to respond
+          await aTimeout(100);
```

Comment:
Would `nextFrame()` work? I noticed that it's used after `_scrollIntoView()` in the following test which is very similar.

Reply:
I tried `nextFrame()` and `aTimeout(0)` but both were flaky

Reply:
To align with the following test, it seems there is one more `nextFrame()` needed. 

```diff
comboBox._scrollIntoView(50);
- // Wait for the async data provider to respond
- await aTimeout(0);
+ await nextFrame()
- // Wait for the __loadingChanged observer
+ // Wait for the async data provider to respond
await aTimeout(0);

...

comboBox._scrollIntoView(0);
+ await nextFrame();
```

I just realized that the result is very close to `aTimeout(100)`. Two `nextFrame()` will give ~ 120ms delay.

Reply:
Feel free to push these changes.

Reply:
I tried to replicate them locally and now the test never passes

Reply:
Hmm. Let me try. It worked for me locally.

Reply:
The revision you pushed seems to work fine and is no longer flaky on FF 👍  lgtm